ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -169,6 +169,10 @@ open class NavigationBar: UINavigationBar, TokenizedControlInternal, TwoLineTitl
     /// A temporary change so that consumers who use SwiftUI for navigation can avoid duplicated resources until support of a swiftUI control is available.
     @objc public static let backButtonAccessibilityLabel: String = "Accessibility.NavigationBar.BackLabel".localized
 
+	/// The accessibility label that should be applied for the done button for when navigation bar is shown in a modal view.
+	/// A temporary change so that consumers who use SwiftUI for navigation can avoid duplicated resources until support of a swiftUI control is available.
+	@objc public static let doneButtonAccessibilityLabel: String = "Accessibility.Done.Label".localized
```

Comment:
since the nav bar doesn't actually use the accessibility label, do we want to export it from a control that does? I'm seeing BarButtonItems?

Reply:
yeah I wasn't sure what the best way to do this was, as the string isn't specific to barbuttonItems per say, its just the only place we use it. 

thinking about when we move to SwiftUI for list, it seems like we will be making our own navigation bar implementation, so it seemed like this would be a good spot given that is where we would have to add it. 

I dont have a strong opinion one way or the other, but grouping it with the other label that had to be exposed seemed also useful